corehq/apps/app_manager/models.py

```diff
@@ -4468,10 +4468,32 @@ def make_build(self, comment=None, user_id=None):
         assert copy._id
         prune_auto_generated_builds.delay(self.domain, self._id)
 
-        self.check_build_dependencies(new_build=copy)
+        self.analyse_app_build(new_build=copy)
 
         return copy
 
+    def analyse_app_build(self, new_build):
+        self.check_for_custom_callouts(new_build)
+        self.check_build_dependencies(new_build)
+
+    def check_for_custom_callouts(self, new_build):
+        from corehq.apps.app_manager.util import app_callout_templates
+
+        templates = next(app_callout_templates)
```

Comment:
should this be list instead of next?

Reply:
The `app_callout_templates` variable holds a reference to a [function which yields](https://github.com/dimagi/commcare-hq/blob/0cf6206de2829c5b157acdfaf062aef9fd662fdf/corehq/apps/app_manager/util.py#L523-L543), so maybe `next` is a better suit here?